docker-compose.circle.yml

```diff
@@ -62,63 +74,78 @@ el6:
 ## Package build nodes
 #
 wheezybuild:
-  image: quay.io/stackstorm/packagingenv:wheezy
+  image: stackstorm/packagingenv:wheezy
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 jessiebuild:
-  image: quay.io/stackstorm/packagingenv:jessie
+  image: stackstorm/packagingenv:jessie
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 trustybuild:
-  image: quay.io/stackstorm/packagingenv:trusty
+  image: stackstorm/packagingenv:trusty
+  extends:
+    file: docker-compose.override.yml
+    service: volumes-circle
+
+xenialbuild:
+  image: stackstorm/packagingenv:xenial
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos6build:
-  image: quay.io/stackstorm/packagingenv:centos6
+  image: stackstorm/packagingenv:centos6
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 centos7build:
-  image: quay.io/stackstorm/packagingenv:centos7
+  image: stackstorm/packagingenv:centos7
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 ## Package testing nodes
 #
 wheezytest:
-  image: quay.io/dennybaa/droneunit:wheezy-sshd
+  image: stackstorm/packagingtest:wheezy
   extends:
     file: docker-compose.override.yml
     service: volumes-circle
 
 jessietest:
-  image: quay.io/dennybaa/droneunit:jessie-sshd
+  image: stackstorm/packagingtest:jessie
```

Comment:
Please use old container naming convention for those who runs with the `init` system or `sshd`.
See this comment: https://github.com/StackStorm/st2-packages/pull/322#discussion_r71505925